comfort-service.yml

```diff
@@ -55,27 +56,54 @@ namespaces:
         description: |
           The position of the entire seat
         members:
-          - name: base
-            datatype: movement_t
+          - name: position
```

Comment:
Aligning names to VSS, e.g. VSS use the term `position` (see https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Cabin/SingleSeat.vspec#L62), so why not use `position`here as well